docs/files/parameter-file.rst

```diff
@@ -3,59 +3,416 @@
+        default: 1.0e-14        :raw-html:`<br />`
+        type: real number :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: tolerance criterion for the convergence of the non-linear flow
+    * random_initial_strain               
+        default: 0.0            :raw-html:`<br />`
+        type: real number :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: non-dimensional value for the initial strain perturbation for the entire domain
+    * pressure_const                      
+        default: -1.0 (i.e. not used) :raw-html:`<br />`
+        type: real number :raw-html:`<br />`
+        unit: Pa :raw-html:`<br />`
+        definition: set constant pressure value for the domain (relevant when 2-D is plain view)
+    * initial_dynamic_range               
+        default: False  :raw-html:`<br />`
+        type: True/False :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: method to smoothen convergence of the velocity field in scenarios with wide viscosity range, see Gerya (2019) :cite:`gerya2019`
+    * periodic_boundary                   
+        default: False  :raw-html:`<br />`
+        type: True/False :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: allows simulation with periodic boundary in the horizontal direction
+    * high_kappa_in_asthenosphere         
+        default: False  :raw-html:`<br />`
+        type: True/False :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: mimics high heat transport in the asthenosphere increasing its thermal diffusivity coefficient
+    * basal_heat                          
+        default: -1.0 (i.e. not used)  :raw-html:`<br />`
+        type: real number :raw-html:`<br />`
+        unit: W/m^2 :raw-html:`<br />`
+        definition: set basal heat flux value
 
-#. Viscosity values
-    
-    * The reference viscosity
-    * The minimum viscosity
-    * The maximum viscosity
+#. Particles options
+
+    * particles_per_element               
+        default: 81           :raw-html:`<br />`
+        type: integer :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: number of Lagrangian particles in each element
+    * particles_per_element_x             
+        default: 0 (automatic calculation) :raw-html:`<br />`
+        type: integer :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: number of Lagrangian particles in the horizontal direction
+    * particles_per_element_z             
+        default: 0 (automatic calculation) :raw-html:`<br />`
+        type: integer :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: number of Lagrangian particles in the vertical direction
+    * particles_perturb_factor            
+        default: 0.5  :raw-html:`<br />`
+        type: real number between 0 and 1 :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: default is 0.5 
```

Comment:
@victorsacek Can you add a description to this parameter? I think we missed it during the discussion